dotCMS/src/main/java/com/dotcms/workflow/form/WorkflowActionForm.java

```diff
@@ -197,6 +173,15 @@ public String getName() {
         @JsonProperty()
         private String        actionCondition;
 
+        @JsonProperty(required = true)
+        private Set<WorkflowStatus> showOn;
+
+
+       public Builder showOn(Set<WorkflowStatus> showOn) {
+           this.showOn = showOn;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "showOn". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)

Reply:
Not needed